### PR TITLE
use steps the way joplin makes them now

### PR DESCRIPTION
**Pull request by briaguya**

### CODE REVIEW

File: src/js/queries/allServicePagesQuery.js [hunk 1]
```diff
@@ -1,65 +1,16 @@
...
-                }
-              }
-            }
-          }
+          ...servicePageInfo
         }
       }
     }
   }
+  ${servicePageFragment}
```
Comment (ifsimicoded) [inline src/js/queries/allServicePagesQuery.js]: 🙏

File: src/js/queries/servicePageFragment.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      id
+      slug
+      text
+      theme {
+        id
+        slug
+        text
+      }
+    }
+    serviceSteps {
```
Comment (ifsimicoded) [inline src/js/queries/servicePageFragment.js]: From the looks of our inbound public health content, it seems like we need a title field and an unknown length of steps (which could include nested lists) here.

File: src/components/Pages/Service.js [hunk 3]
```diff
@@ -50,11 +50,11 @@ const Service = ({
     <div className="wrapper wrapper--sm container-fluid">
       <PageHeader>{title}</PageHeader>
 
-      {steps &&
-        !!steps.length && (
+      {serviceSteps &&
+        !!serviceSteps.edges.length && (
```
Comment (ifsimicoded) [inline src/components/Pages/Service.js]: Can we remove edges when we process this data in static.config.js -- I think our components should be graphql neutral. We have other clean functions that set this pattern.

File: src/js/queries/servicePageFragment.js (hunk 2 above)
Comment (briaguya) [reply src/js/queries/servicePageFragment.js]: I noticed the need for a title [when updating the yaml in joplin]((https://github.com/cityofaustin/joplin/pull/155/files#diff-9c5ea5128d161862f6213b2beb0cf3f2R12). Could you clarify what you mean by an unknown length of steps?

**Pull request merged**